src/profile/public/main.js

```diff
@@ -134,7 +109,7 @@ Promise.all([
 
 
 	// Handle login form.
-	var keyPair;
+	var keyPair = JSON.parse(localStorage.getItem('dfinity-ic-user-identity'));
```

Comment:
I really don't want to do this. I'd much prefer the UserLib had something like `ic.getKeyPair`).

Reply:
I want to move HttpAgent to window.ic.agent and have more functions in the ic namespace. I want to do a proper review and design of our API after bronze